sphinx_gallery/gen_rst.py

```diff
@@ -260,7 +261,7 @@ def save_figures(image_path, fig_count, gallery_conf):
             figure_list.append(current_fig)
         mlab.close(all=True)
 
-    return figure_rst(figure_list, gallery_conf['src_dir'])
+    return figure_rst(figure_list, gallery_conf['src_dir']), len(figure_list)
```

Comment:
This definitely feels nice to simplify the return type.

Reply:
I'm not sure to get the point of this comment.

Reply:
Just a compliment, maybe you are not used to that in a review ;-)